client/src/app/site/users/services/user-import.service.ts

```diff
@@ -36,8 +36,8 @@ export class UserImportService extends BaseImportService<User> {
         'default_password',
         'email',
         'username',
-        'gender',
-        'vote_weight'
```

Comment:
you can query the default values. Adding the template fields  is a todo, right?

Reply:
I didn't change it there, because of #92. The views to import users, motions, etc. will be updated.